src/schemas/21P-530/schema.js

```diff
@@ -124,7 +119,16 @@ let schema = {
     },
     amountGovtContribution: {
       type: 'number'
-    }
+    },
+    claimantEmail: {
+      type: 'string',
+      format: 'email'
+    },
+    'view:claimantEmailConfirmation': {
```

Comment:
i think the view fields are supposed to be used only in on the frontend and removed before sending to the backend, you can add this field after importing the schema and do the email check entirely in the frontend